haystack/components/routers/conditional_router.py

```diff
@@ -113,12 +117,19 @@ def __init__(self, routes: List[Dict]):
             - `output_type`: The type of the output data (e.g., str, List[int]).
             - `output_name`: The name under which the `output` value of the route is published. This name is used to
                 connect the router to other components in the pipeline.
+        :param custom_filters: A dictionary of custom Jinja2 filters to be used in the condition expressions.
+            For example, passing `{"my_filter": my_filter_fcn}` where:
+            - `my_filter` is the name of the custom filter.
+            - `my_filter_fcn` is a callable that takes a variable and returns a value.
```

Comment:
Let's explicitly describe the parameters and output of the callable.

Reply:
Updated docstring